<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogSetting</class>
 <widget class="QWidget" name="LogSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日志开关</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_main">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_settings" stretch="0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_group">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_group_heading">
           <item>
            <widget class="QLabel" name="label_group">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Group</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_group_reset">
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_group_select_all">
             <property name="text">
              <string>全选</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_current_group">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Current:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="listView_group">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_mask">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_mask_heading">
           <item>
            <widget class="QLabel" name="label_mask">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Mask</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_mask_reset">
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_mask_select_all">
             <property name="text">
              <string>全选</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_search">
           <item>
            <widget class="QLineEdit" name="lineEdit_mask_search">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>请输入完整mask</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_mask_search">
             <property name="text">
              <string>搜索</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="listView_mask"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_check">
       <item>
        <widget class="QCheckBox" name="checkBox_systemlog">
         <property name="accessibleName">
          <string>System Log</string>
         </property>
         <property name="text">
          <string>System Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_offlinelog">
         <property name="accessibleName">
          <string>Offline Log</string>
         </property>
         <property name="text">
          <string>Offline Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_drqlog">
         <property name="accessibleName">
          <string>DRQ Log</string>
         </property>
         <property name="text">
          <string>DRQ Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_metadatalog">
         <property name="accessibleName">
          <string>Metadata Log</string>
         </property>
         <property name="text">
          <string>Metadata Log</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_preview">
       <property name="text">
        <string>预览</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listView_preview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_buttons">
       <item>
        <widget class="QPushButton" name="pushButton_load">
         <property name="text">
          <string>读取设备预设</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_clear">
         <property name="text">
          <string>清除设备预设</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_buttons">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_reset">
         <property name="text">
          <string>恢复默认值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_apply">
         <property name="text">
          <string>应用</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
